libs/core-ui/src/lib/Highchart/getDefaultHighchartOptions.ts

```diff
@@ -34,6 +34,20 @@ export function getDefaultHighchartOptions(theme: ITheme): Highcharts.Options {
       zoomType: "xy"
     },
     credits: undefined,
+    exporting: {
+      buttons: {
+        contextButton: {
+          menuItems: [
+            "viewFullscreen",
+            "printChart",
+            "separator",
+            "downloadPNG",
+            "downloadJPEG",
+            "downloadPDF"
```

Comment:
Can we add a test for testing the download of different file types? It this seems out of scope could we create an issue to track this?

Reply:
Thanks for the comment. I created a task and assigned to myself: https://msdata.visualstudio.com/Vienna/_workitems/edit/1830300
Will work on it later